packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -27,6 +27,12 @@ const KEY_UP = 'ArrowUp';
 const KEY_DOWN = 'ArrowDown';
 const KEY_TAB = 'Tab';
 
+// Configuration default
+const DEFAULT_CONFIG = {
+  renderTags: true, // Allow the Multiselect to generate the Tag elements in the DOM
```

Comment:
In what case would we want this to be false? Is this to support https://www.consumerfinance.gov/data-research/prepaid-accounts/search-agreements/ ? Could we maybe change that to show tags with the multiselect?

Reply:
@anselmbradford This option was added to allow for a workaround (we render the Tags ourselves) to a bug we encountered while trying to integrate the DS Multiselect into the `design-system-react` implementation of the DS.  The DS icons are not rendering correctly. 
 
I've tried to debug but have yet to find a resolution. 

|DS Rendered Tags|React rendered Tags|
|---|---|
|<img width="980" alt="Screenshot 2023-11-13 at 11 01 21 AM" src="https://github.com/cfpb/design-system/assets/2592907/15e5164c-4299-43d9-83e7-ff558f177b9a">|![Screenshot 2023-11-13 at 11 08 25 AM](https://github.com/cfpb/design-system/assets/2592907/187af02c-cfab-437b-b38a-ec6142a5b98d)|

Reply:
Maybe add a `TODO: ` comment above this explaining why it's needed.